std_msgs/String emotion
std_msgs/Float32 intensity